src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,130 @@
+  },
+  {
+    forumId: 'Yellowdig1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Yellowdig is the digital solution that impacts the entire student lifecycle and enables lifelong learning.',
+    supportLevel: 'Coming soon',
+    isAvailable: false,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'edX Forum2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
+    supportLevel: 'Full support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Piazza2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
+    supportLevel: 'Partial support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+];
+
+function chunk(arr, len) {
+  const chunks = [];
+  let i = 0;
+  const n = arr.length;
+  while (i < n) {
+    chunks.push(arr.slice(i, i += len));
+  }
+  return chunks;
+}
+
+function ForumToolSelectorSettings({ intl }) {
+  const [selectedForumId, setSelectedForumId] = useState(null);
+
+  const onSelect = (forumId) => {
+    setSelectedForumId(forumId);
+  };
+
+  return (
+    <div className="container-fluid text-info-300">
+      <h6 className="mt-3 text-center">{intl.formatMessage(messages.heading)}</h6>
+
+      {chunk(allForums, 3).map(forums => (
+        <div className="mt-4" key={forums[0].forumId}>
+          <div className="card-deck row">
+            {forums.map(forum => (
+              <ForumCard
+                key={forum.forumId}
+                forum={forum}
+                selected={forum.forumId === selectedForumId}
+                onSelect={onSelect}
+              />
+            ))}
+          </div>
+
+          <h2 className="mt-3 mb-3">{intl.formatMessage(messages.supportedFeatures)}</h2>
+
+          <FeaturesTable forums={forums} />
+        </div>
+      ))}
+    </div>
+  );
+}
+
+ForumToolSelectorSettings.propTypes = {
+  intl: intlShape.isRequired,
+};
+
+export default injectIntl(ForumToolSelectorSettings);
```

Comment:
Can you rename this component to DiscussionToolSelector? I don't see the need for the `Settings` and we've generally referred to them as discussion tools.